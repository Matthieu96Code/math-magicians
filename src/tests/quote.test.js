import React from 'react';
import renderer from 'react-test-renderer';
import Quotes from '../components/displayQuote';

it('renders correctly', () => {
  const component = renderer.create(
    <Quotes />,
  );
  const tree = component.toJSON();

  expect(tree).toMatchSnapshot();
});
