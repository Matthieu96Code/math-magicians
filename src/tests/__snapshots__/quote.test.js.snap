// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="quotePage"
>
  <p
    className="quote"
  >
    loading ...
  </p>
</div>
`;
